feathr-example/bureau_bureau-balance.ipynb

```diff
@@ -0,0 +1,1224 @@
+      "      f_SK_ID_CURR f_SK_ID_BUREAU f_NUM_CREDIT_COUNT f_DEBT_CREDIT_RATIO\n",
+      "0           298636        5000047                 13                None\n",
+      "1           136548        5000433                  2            0.970875\n",
+      "2           162526        5000779                  7                 0.0\n",
+      "3           426444        5000855                 19                None\n",
+      "4           221038        5001087                  3                 1.0\n",
+      "...            ...            ...                ...                 ...\n",
+      "8562        197548        6841943                 25                 0.0\n",
+      "8563        210960        6842052                  1                 0.0\n",
+      "8564        187591        6842344                 15                 0.0\n",
+      "8565        242758        6843074                  5                 0.0\n",
+      "8566        130212        6843220                  8  0.9631681957186544\n",
+      "\n",
+      "[1716428 rows x 4 columns]\n"
+     ]
+    }
+   ],
+   "source": [
+    "\n",
+    "with pd.option_context('display.max_columns', 50, 'display.max_rows', 1000):\n",
+    "   print(df_res.columns.values.tolist())\n",
+    "   print(df_res[[\n",
+    "      \"f_SK_ID_CURR\",\n",
+    "      \"f_SK_ID_BUREAU\",\n",
+    "      \"f_CREDIT_ACTIVE\",\n",
+    "      \"f_CREDIT_CURRENCY\",\n",
+    "      \"f_DAYS_CREDIT\",\n",
+    "      \"f_CREDIT_DAY_OVERDUE\",\n",
+    "      \"f_DAYS_CREDIT_ENDDATE\",\n",
+    "      \"f_DAYS_ENDDATE_FACT\",\n",
+    "      \"f_AMT_CREDIT_MAX_OVERDUE\",\n",
+    "      \"f_CNT_CREDIT_PROLONG\",\n",
+    "      \"f_AMT_CREDIT_SUM\",\n",
+    "      \"f_AMT_CREDIT_SUM_DEBT\",\n",
+    "      \"f_AMT_CREDIT_SUM_LIMIT\",\n",
+    "      \"f_AMT_CREDIT_SUM_OVERDUE\",\n",
+    "      \"f_CREDIT_TYPE\",\n",
+    "      \"f_DAYS_CREDIT_UPDATE\",\n",
+    "      \"f_AMT_ANNUITY\",\n",
+    "\n",
+    "      \"f_SK_ID_BUREAU\",\n",
+    "      \"f_MONTHS_BALANCE\",\n",
+    "      \"f_STATUS\",\n",
+    "      \"f_CREDIT_STATUS_EMA_AVG_DER\",\n",
+    "      \"f_CREDIT_STATUS_EMA_AVG\",\n",
+    "\n",
+    "      \"f_NUM_CREDIT_COUNT\",\n",
+    "      \"f_DEBT_CREDIT_RATIO\"\n",
+    "   ]])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2022-09-02 12:03:35.769 | INFO     | feathr.spark_provider._synapse_submission:upload_or_get_cloud_path:66 - Uploading /var/folders/gs/dbrzk90d0m3849n982_q27w40000gn/T/tmpflkjpany/feathr_pyspark_driver.py to cloud..\n",
```

Comment:
Logs

Reply:
Cleaned up the outputs of the notebooks.